templates/AddressMap.ss

```diff
@@ -1,5 +1,5 @@
 <div class="addressMap">
 	<a href="//maps.google.com/?q=$Address">
-		<img src="//maps.googleapis.com/maps/api/staticmap?size={$Width}x{$Height}&markers=$Address&sensor=false" alt="$FullAddress.ATT" />
```

Comment:
My only remaining question with this is how removing `sensor` affects the map should someone already be using it? Other than that, this looks good to merge through.

Reply:
@tardinha, just following up to make sure I can get this merged through for you.

Reply:
@nglasl Sensor has been removed: https://developers.google.com/maps/documentation/static-maps/intro#Moreinfo :)

Reply:
Ah, well that makes sense then :)